client/gtk2/ibusimcontext.c

```diff
@@ -381,10 +381,10 @@ typedef struct {
 } ProcessKeyEventData;
 
 typedef struct {
-    int       count;
-    guint     count_cb_id;
-    gboolean  retval;
-} ProcessKeyEventReplyData;
+    GAsyncResult *res;
+    GMainContext *context;
```

Comment:
Maybe good to replace context with gcontext not to confuse with IBusInputContext?

Reply:
Right, I can use another name for this variable like main_context.